plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/meta/CubridMetaModel.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ext.cubrid.model.meta;
+
+import java.sql.SQLException;
+
+import org.jkiss.code.NotNull;
+import org.jkiss.code.Nullable;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.ext.cubrid.CubridConstants;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridObjectContainer;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridTable;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridTableBase;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridTableIndex;
+import org.jkiss.dbeaver.ext.cubrid.model.CubridView;
+import org.jkiss.dbeaver.ext.generic.model.GenericStructContainer;
+import org.jkiss.dbeaver.ext.generic.model.GenericTableBase;
+import org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel;
+import org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaObject;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCResultSet;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCSession;
+import org.jkiss.dbeaver.model.exec.jdbc.JDBCStatement;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCConstants;
+import org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils;
+import org.jkiss.dbeaver.model.struct.rdb.DBSIndexType;
+
+public class CubridMetaModel extends GenericMetaModel {
+	private static final Log log = Log.getLog(CubridMetaModel.class);
+
+	public CubridMetaModel()
+	{
+	}
+
+	public boolean isSupportMultiSchema(JDBCSession session) {
+		try {
+			int major = session.getMetaData().getDatabaseMajorVersion();
+			int minor = session.getMetaData().getDatabaseMinorVersion();
+			if(major == 11 && minor >= 2) {
+				return true;
+			}else if(major > 11) {
+				return true;
+			}
+		} catch (SQLException e) {
+			log.error("can't get database version");
+		}

```

Comment:
Formatting issues and exception should passed as `log.error` argument

Reply:
Also you can concatenate condition and make `major > 11 || (major == 11 && minor >= 2)` instead of two similar if conditions

Reply:
I have fixed it.